compiler/src/iree/compiler/Codegen/LLVMCPU/KernelDispatch.cpp

```diff
@@ -92,6 +92,12 @@ static llvm::cl::opt<bool> clEnableScalableVectorization(
                    "target (e.g., +sve, +sve2 and/or +sme feature flags)"),
     llvm::cl::init(false));
 
+static llvm::cl::opt<bool> clDisableArmSMETiling(
```

Comment:
I think that we should keep this as experimental and keep the original comment:
> // NOTE: This flag is meant for testing + experimentation and should not be
// used in deployment.